resources.go

```diff
@@ -1229,6 +1229,15 @@ func Provider() tfbridge.ProviderInfo {
 				"Pulumi":                       "1.5.0-preview",
 				"System.Collections.Immutable": "1.6.0",
 			},
+			Overlay: &tfbridge.OverlayInfo{
+				Modules: map[string]*tfbridge.OverlayInfo{
+					"Storage": {
+						DestFiles: []string{
+							"SharedAccessSignature.cs",
```

Comment:
I'm not even sure if we need this overlay, as tfgen doesn't have to do anything with it. Anyway, it seems reasonable to have a list of mixin files here.